source/docs/advanced/payments/adyen.md

```diff
@@ -24,6 +24,7 @@ P.S. No Magento modules are needed if you use this method. If you however want t
 - `FRONT_COMMERCE_ADYEN_NOTIFICATION_PASSWORD` (more on [webhooks below)](#Add-webhook) you create it. You then have to copy it to the webhook section in your Adyen Customer Area under `Developers > Webhooks > [your_prefered_webhook] > Authentication > Password`
 - `FRONT_COMMERCE_ADYEN_HMAC_KEY` (more on [webhooks below](#Add-webhook)) create it from the webhook section in your Adyen Customer Area under `Developers > Webhooks > [your_prefered_webhook] > Additional Settings > HMAC key`
 - `FRONT_COMMERCE_ADYEN_PREVIOUS_HMAC_KEY` (more on [webhooks below](#Add-webhook)) when you regenerate your HMAC key configure this to be the old one
+- `FRONT_COMMERCE_ADYEN_STORE_DETAILS` Boolean (more on [Adyen's Stored Payment Methods](#Enable-Adyen’s-Stored-Payment-Methods-optional-1)). If you want to enable Adyen's Stored Payment Methods (Tokenization).
```

Comment:
I didn't realized on the merge request, but the variable name seems a bit weird, why not `FRONT_COMMERCE_ADYEN_STORE_PAYMENT_METHODS` ?

Reply:
I thought about this as well, but the feature isn't really about storing a payment method, but storing details of a specific one (in the case of Adyen, it's specific to credit cards), so I found it was a bit of a "faux-ami".

Also in the code (and calls), they mix `storeDetails` and `storedPaymentMethods` so that's a bit of bias on my end I guess.

Reply:
Should I make a quick MR to fix this?

Reply:
It does not seem critical either, @paul-vd @real34, MR or not MR ?

Reply:
I also agree with the sense of having this specify that it's related to payment so `FRONT_COMMERCE_ADYEN_STORE_PAYMENT_DETAILS`

> Should I make a quick MR to fix this?

@Kagaminara I think so, it's better to do it before we do a release to avoid a break change.

Reply:
Done [here](https://github.com/front-commerce/developers.front-commerce.com/pull/430)